istioctl/cmd/workload.go

```diff
@@ -364,7 +366,17 @@ func createMeshConfig(kubeClient kube.ExtendedClient, wg *clientv1alpha3.Workloa
 	if revision != "" {
 		istioCM = fmt.Sprintf("%s-%s", istioCM, revision)
 	}
-	istio, err := kubeClient.CoreV1().ConfigMaps(istioNamespace).Get(context.Background(), istioCM, metav1.GetOptions{})
+	istio := &v1.ConfigMap{}
+	err := retry.UntilSuccess(func() error{
+		var err error
+		istio, err = kubeClient.CoreV1().ConfigMaps(istioNamespace).Get(context.Background(), istioCM, metav1.GetOptions{})
+		if err != nil {
+			fmt.Println("configmap istio-ca-root-cert not found, retry waiting")
+			return fmt.Errorf("configmap %s was not found in namespace %s: %v", controller.CACertNamespaceConfigMap, wg.Namespace, err)
+		}
+		return nil
+	}, retry.Timeout(5*time.Minute))
```

Comment:
Would it make sense for `istioctl x workload entry configure` to have a `--timeout duration` flag similar to `istioctl x wait`?